libnavigation-base/src/main/java/com/mapbox/navigation/base/routerefresh/RouteRefreshAdapterProvider.kt

```diff
@@ -0,0 +1,11 @@
+package com.mapbox.navigation.base.routerefresh
+
+object RouteRefreshAdapterProvider {
```

Comment:
Building on comment here https://github.com/mapbox/mapbox-navigation-android/pull/3184/files#r441160524

We don't want to make this an object. We'll run into similar issues that we have from the ThreadController
https://github.com/mapbox/mapbox-navigation-android/issues/3156